CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## v0.2.1 - 2020-06-24
+
+This is a patch release.
```

Comment:
I wasn't sure what more to put here :smile:

Reply:
I suggest mentioning the main purpose of the release

```suggestion
This is a patch release to fix AKS platform deployments.
```